torch/_C/__init__.pyi.in

```diff
@@ -1350,3 +1350,4 @@ def _enable_minidumps(directory: str) -> None: ...
 def _disable_minidumps() -> None: ...
 def _enable_minidumps_on_exceptions() -> None: ...
 def _register_py_class_for_device(device: str, cls: Any) -> None: ...
+def _activate_cuda_trace() -> None: ...
```

Comment:
Please add a newline at the end of the file. There is a setting in vscode (if you're using that) to do this automatically. And I guess other editors have it too.

Reply:
Sorry, fixed.